apps/server/src/modules/account/mapper/account-entity-to-dto.mapper.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { Account } from '@shared/domain/entity';
-import { ObjectId } from 'bson';
+import { ObjectId } from '@mikro-orm/mongodb';
```

Comment:
Why is it important to import `ObjectId` from `@micro-orm/mongodb` and not from `bson`? Do we have to use the specific `bson` version that is used by MikroORM?
https://github.com/mikro-orm/mikro-orm/blob/8799461ddf05637c1d227e3c7879c99b9bb74bd2/packages/mongodb/src/index.ts#L15

Reply:
Yes,mikroORM checks which version of bson has created the objectId. It has to be the same version as mikroORM uses.